twirl-compiler/src/main/scala/CommandLineTwirlTemplateCompiler.scala

```diff
@@ -1,61 +1,67 @@
 package rulestwirl.twirl
 
 import higherkindness.rules_scala.common.error.AnnexWorkerError
+import higherkindness.rules_scala.common.sandbox.SandboxUtil
 import higherkindness.rules_scala.common.worker.WorkerMain
 import play.twirl.compiler.TwirlCompiler
 import java.io.{File, PrintStream}
-import java.nio.file.{Files, Paths}
-import scala.collection.JavaConverters._
+import java.nio.file.{Files, Path, Paths}
+import scala.jdk.CollectionConverters._
 import scala.util.boundary, boundary.break
 import scopt.OParser
 
 object CommandLineTwirlTemplateCompiler extends WorkerMain[Unit] {
 
   case class Config(
-    additionalImports: Seq[String] = Seq.empty[String],
-    source: File = new File("."),
-    sourceDirectory: File = new File("."),
-    templateFormats: Map[String, String] = Map.empty[String, String],
-    output: File = new File("."),
+    var additionalImports: List[String] = List.empty[String],
+    var source: Path = Paths.get("."),
+    var sourceDirectory: Path = Paths.get("."),
+    var templateFormats: Map[String, String] = Map.empty[String, String],
+    var output: Path = Paths.get("."),
   )
 
   val builder = OParser.builder[Config]
-  val parser = {
+  def parser(workDir: Path) = {
     import builder._
     OParser.sequence(
       programName("twirl-compiler"),
       head("Twirl Template Compiler", "0.2"),
-      arg[File]("<output>").required().action { (value, config) =>
-        config.copy(output = value)
+      arg[Path]("<output>").required().action { (value, config) =>
+        config.output = SandboxUtil.getSandboxPath(workDir, value)
```

Comment:
Nit: I'm not a huge fan of the mutability here. What's wrong with `.copy`?

Reply:
For completeness I'm copying my comment from the play routes cli PR:

Speed is the reason. I do think it's worth it because that many object copies is not very fast.

Also I'm not too worried about it as the only uses of this object are in this file, it's very short, and reasoning about correctness is much easier than a larger codebase.